### PR TITLE
Update the docs

### PR DESCRIPTION
**Pull request by Motorenger**
Corresponding to #98 
- Updated the docs to the current command for building, installing...
- Fix README
- Replace MANIFEST.in

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -13,7 +13,8 @@ Python XMP Toolkit
 ----------------------
 The short version of installation is::
 
-  python setup.py install
```
Comment (zougloub) [inline INSTALL:16]: Shouldn't `pip install` be the preferred entry point?

File: docs/developers.rst [hunk 2]
```diff
@@ -46,13 +46,13 @@ For test coverage, run::
 
 Distribution Configuration
 --------------------------
-The file ``setup.py`` specify how the distribution is packed together. Most
+The file ``pyproject.toml`` specify how the distribution is packed together. Most
```
Comment (zougloub) [inline docs/developers.rst:49]: You can correct to "specifies" also.

File: INSTALL (hunk 1 above)
Comment (Motorenger) [reply INSTALL:16]: OK

File: docs/developers.rst (hunk 2 above)
Comment (Motorenger) [reply docs/developers.rst:49]: OK

File: docs/developers.rst [hunk 3]
```diff
@@ -46,13 +46,13 @@ For test coverage, run::
...
 important to note is that version in formation is read from ``libxmp.version``
 file, and that the file ``MANIFEST.in`` specifies which other files to include
```
[the remark this answers is not in the document]
Comment (Motorenger) [reply docs/developers.rst:51]: Flit indeed doesn't support `MANIFEST.in` at all, so I got rid of it and replaced it's functionality in the `pyproject.toml`

File: INSTALL [hunk 4]
```diff
@@ -13,11 +13,12 @@ Python XMP Toolkit
...
+  pip install python-xmp-toolkit
 
-or if you use pip::
+or if you use flit::
```
Comment (zougloub) [inline INSTALL:18]: I think this "or" is useless, because today everybody uses pip

File: docs/developers.rst [hunk 5]
```diff
@@ -11,7 +11,7 @@ Overview of Source Distribution
 
  * ``docs/`` -- Source code for documentation.
  * ``libxmp/`` -- Source files for XMP Toolkit
- * ``setup.py`` -- Distutils configuration file.
+ * ``pyproject.toml`` -- Flit configuration file.
```
Comment (zougloub) [inline docs/developers.rst:14]: This is not specific to flit, so there's no need to specify that flit is used. This is a packaging cofniguration file.

File: docs/developers.rst [hunk 6]
```diff
@@ -22,13 +22,13 @@ http://sphinx.pocoo.org/). To make the documentation run the following command
 in the root directory::
 
   pip install sphinx
-  sphinx-build -M html docs/ docs/_build/
+  sphinx-build docs/ docs/_build/
 
 Packaging a Distribution
```
Comment (zougloub) [inline docs/developers.rst:27]: I think this section is pointless and we should replace it with "publishing a release" and indicate that in order to publish to pypi, follow instructions in https://flit.pypa.io/en/stable/

 (pending more details).

File: INSTALL (hunk 4 above)
Comment (Motorenger) [reply INSTALL:18]: OK

File: docs/developers.rst (hunk 5 above)
Comment (Motorenger) [reply docs/developers.rst:14]: OK

**Pull request merged**